flask.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Flask
+layout: 2017/sheet
+category: Flask  # could be split later on
+updated: 2018-04-06
+prism_languages: [python, bash, django]
+intro: |
+  Flask is a microframework for Python based on Werkzeug, Jinja 2 and good intentions.
+tags:
+  - WIP
+  - Python
+---
+
+{% raw %}
+
+Flask
+-----
+
+## Configuration
```

Comment:
an `h2` followed by an `h2` seems strange, but you probably did this to make the "Configuration" heading visible. may I suggest filling in the "Flask" heading with a super-quick getting started guide / hello world?

eg: https://devhints.io/fastify

Reply:
Will do!